httpclient-backend/monix/src/test/scala/sttp/client/httpclient/monix/HttpClientHighLevelMonixWebsocketTest.scala

```diff
@@ -22,12 +22,12 @@ class HttpClientHighLevelMonixWebsocketTest extends HighLevelWebsocketTest[Task,
   implicit val convertToFuture: ConvertToFuture[Task] = convertMonixTaskToFuture
   implicit val monad: MonadError[Task] = TaskMonadAsyncError
 
-  def createHandler: Option[Int] => Task[WebSocketHandler[WebSocket[Task]]] = _ => MonixWebSocketHandler(5)
+  def createHandler: Option[Int] => Task[WebSocketHandler[WebSocket[Task]]] = _ => MonixWebSocketHandler()
```

Comment:
I guess the argument doesn't make sense? :)

Reply:
no it doesn't :P

Reply:
But it's an overridden method from `HighLevelWebSocketTest`. I'll add the `override` keyword

Reply:
the alternative would be to provide an empty implementation, and e.g. create the handler inline:
```  override def createHandler: Option[Int] => Task[WebSocketHandler[WebSocket[Task]]] = ???

  it should "handle backpressure correctly" in {
    basicRequest
      .get(uri"$wsEndpoint/ws/echo")
      .openWebsocketF(MonixWebSocketHandler())
//(...)
```
Which one do you think would be better?

Reply:
Ah, if it's an override, don't bother, it's good as it is :)